client/my-sites/media-library/style.scss

```diff
@@ -115,3 +115,27 @@
 .media-library__list-item.is-small .media-library__list-item-file-extension {
 	display: none;
 }
+
+.media-library__videopress-nudge-container {
+	display: block;
+	overflow-y: auto;
+	position: absolute;
+	top: 50px;
+	bottom: 72px;
+	right: 0;
+	left: 0;
+	margin: 0;
+}
+
+.media-library__videopress-nudge-container .upgrade-nudge-expanded {
+	position: absolute;
+	left: 0;
+	right: 0;
+	box-shadow: none;
+	margin: 0;
+
+	@include breakpoint( ">960px") {
+		top: 50%;
+		transform: translateY(-50%);
+	}
```

Comment:
This is just my opinion, but I really don't care for the vertical centering we're trying to do in wider breakpoints. I'd rather just see it tied to the top of the page with some padding.